samples/nrf9160/mqtt_simple/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2018 Nordic Semiconductor
+# Copyright (c) 2020 Nordic Semiconductor
```

Comment:
There's no need to update the year here, there were no modifications in this file.